scripts/in_container/prod/entrypoint_prod.sh

```diff
@@ -109,51 +109,116 @@ function verify_db_connection {
-elif [[ ${AIRFLOW_COMMAND} == "python" ]]; then
-   shift
-   exec "python" "${@}"
-elif [[ ${AIRFLOW_COMMAND} == "airflow" ]]; then
-   AIRFLOW_COMMAND="${2}"
-   shift
-fi
+function create_system_user_if_missing() {
+    # This is needed in case of OpenShift-compatible container execution. In case of OpenShift random
+    # User id is used when starting the image, however group 0 is kept as the user group. Our production
+    # Image is OpenShift compatible, so all permissions on all folders are set so that 0 group can exercise
+    # the same privileges as the default "airflow" user, this code checks if the user is already
+    # present in /etc/passwd and will create the system user dynamically, including setting its
+    # HOME directory to the /home/airflow so that (for example) the ${HOME}/.local folder where airflow is
+    # Installed can be automatically added to PYTHONPATH
+    if ! whoami &> /dev/null; then
+      if [[ -w /etc/passwd ]]; then
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${AIRFLOW_USER_HOME_DIR}:/sbin/nologin" \
+            >> /etc/passwd
+      fi
+      export HOME="${AIRFLOW_USER_HOME_DIR}"
+    fi
+}
 
-# Note: the broker backend configuration concerns only a subset of Airflow components
-if [[ ${AIRFLOW_COMMAND} =~ ^(scheduler|celery|worker|flower)$ ]]; then
-    if [[ -n "${AIRFLOW__CELERY__BROKER_URL_CMD=}" ]]; then
-        verify_db_connection "$(eval "$AIRFLOW__CELERY__BROKER_URL_CMD")"
+function verify_connections() {
+    if [[ -n "${AIRFLOW__CORE__SQL_ALCHEMY_CONN_CMD=}" ]]; then
+        verify_db_connection "$(eval "${AIRFLOW__CORE__SQL_ALCHEMY_CONN_CMD}")"
     else
-        AIRFLOW__CELERY__BROKER_URL=${AIRFLOW__CELERY__BROKER_URL:=}
-        if [[ -n ${AIRFLOW__CELERY__BROKER_URL=} ]]; then
-            verify_db_connection "${AIRFLOW__CELERY__BROKER_URL}"
+        # if no DB configured - use sqlite db by default
+        AIRFLOW__CORE__SQL_ALCHEMY_CONN="${AIRFLOW__CORE__SQL_ALCHEMY_CONN:="sqlite:///${AIRFLOW_HOME}/airflow.db"}"
+        verify_db_connection "${AIRFLOW__CORE__SQL_ALCHEMY_CONN}"
+    fi
+
+}
+
+function upgrade_db() {
+    airflow db upgrade || airflow upgradedb || true
+}
+
+function verify_celery_connection_if_needed() {
+    # Note: the broker backend configuration concerns only a subset of Airflow components
+    if [[ ${AIRFLOW_COMMAND} =~ ^(scheduler|celery|worker|flower)$ ]]; then
+        if [[ -n "${AIRFLOW__CELERY__BROKER_URL_CMD=}" ]]; then
+            verify_db_connection "$(eval "${AIRFLOW__CELERY__BROKER_URL_CMD}")"
+        else
+            AIRFLOW__CELERY__BROKER_URL=${AIRFLOW__CELERY__BROKER_URL:=}
+            if [[ -n ${AIRFLOW__CELERY__BROKER_URL=} ]]; then
+                verify_db_connection "${AIRFLOW__CELERY__BROKER_URL}"
+            fi
         fi
     fi
+}
+
+function execute_bash_or_python_command_if_specified() {
+    # The Bash and python commands still should verify the basic connections so they are run after the
```

Comment:
```suggestion
    # The Bash and Python commands still should verify the basic connections so they are run after the
```

Reply:
addded `bash` and `python` instead as those are commands to use when running the image.